apps/omg/test/omg/state/transaction_test.exs

```diff
@@ -56,8 +56,8 @@ defmodule OMG.State.TransactionTest do
 
     test "raw transaction hash is invariant" do
       assert Transaction.raw_txhash(@transaction) ==
-               <<22, 144, 106, 111, 111, 175, 135, 22, 122, 230, 103, 220, 153, 161, 147, 239, 150, 205, 18, 226, 161,
-                 163, 127, 20, 163, 205, 24, 68, 0, 213, 103, 161>>
+               <<32, 195, 156, 58, 16, 182, 26, 126, 67, 146, 125, 172, 69, 109, 119, 124, 233, 106, 160, 244, 41, 62,
```

Comment:
same as above, is that because a normal payment type transaction now consist of more fields then it did before ALD?

Reply:
yes, same reason and more specifically: addition of `output_type` field in the output

Reply:
another noob question. why do we need `output_type` in the output hash?

Reply:
because a transaction referencing different `output_types` might mean a different thing and should thereby be included in the signed struct hash.